### PR TITLE
Added note for trying out WebAssembly sandbox

### PR DESCRIPTION
**Pull request by gargnupur**
Added note for trying out WebAssembly sandbox for metadata  exchange filter

Please provide a description for what this PR is for.
This is to help users tryout wasm for mixerless telemetry

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ X] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/ops/telemetry/in-proxy-service-telemetry/index.md [hunk 1]
```diff
@@ -38,6 +38,14 @@ To generate service-level metrics directly in the Envoy proxies, follow these st
     $ kubectl -n istio-system apply -f https://raw.githubusercontent.com/istio/proxy/master/extensions/stats/testdata/istio/metadata-exchange_filter.yaml
     {{< /text >}}
 
+    or
+
+    Enable metadata exchange filter with WebAssembly sandbox using the following command:
+
+    {{< text bash >}}
+    $ kubectl -n istio-system apply -f https://raw.githubusercontent.com/istio/proxy/master/extensions/stats/testdata/istio/metadata-exchange_with_wasm_filter.yaml
```
Comment (PiotrSikora) [inline content/en/docs/ops/telemetry/in-proxy-service-telemetry/index.md]: This file won't be available in `master` branch of istio/proxy, will it?

Comment (gargnupur) [reply content/en/docs/ops/telemetry/in-proxy-service-telemetry/index.md]: no it won't

Comment (gargnupur) [reply content/en/docs/ops/telemetry/in-proxy-service-telemetry/index.md]: Updated the link

Comment (gargnupur) [reply content/en/docs/ops/telemetry/in-proxy-service-telemetry/index.md]: @PiotrSikora : PR is ready for review!

**Comment by PiotrSikora** [comment]
Blocked on https://github.com/istio/proxy/pull/2433

File: content/en/docs/ops/telemetry/in-proxy-service-telemetry/index.md (hunk 1 above)
Comment (PiotrSikora) [reply content/en/docs/ops/telemetry/in-proxy-service-telemetry/index.md]: Looks good, but we cannot merge it before https://github.com/istio/proxy/pull/2433.

File: content/en/docs/ops/telemetry/in-proxy-service-telemetry/index.md [hunk 2]
```diff
@@ -38,6 +38,16 @@ To generate service-level metrics directly in the Envoy proxies, follow these st
     $ kubectl -n istio-system apply -f https://raw.githubusercontent.com/istio/proxy/master/extensions/stats/testdata/istio/metadata-exchange_filter.yaml
     {{< /text >}}
 
+    {{< tip >}}
+    Alternatively, if you have Istio 1.3.x, you can enable metadata exchange filter with WebAssembly sandbox using the
+    following command:
+
+    {{< text bash >}}
+    $ kubectl -n istio-system apply -f https://raw.githubusercontent.com/istio/proxy/release-1.3/extensions/stats/testdata/istio/metadata-exchange_with_wasm_filter.yaml
```
Comment (geeknoid) [inline content/en/docs/ops/telemetry/in-proxy-service-telemetry/index.md]:
```suggestion
    $ kubectl -n istio-system apply -f https://raw.githubusercontent.com/istio/proxy/{{< source_branch_name >}}/extensions/stats/testdata/istio/metadata-exchange_with_wasm_filter.yaml
```

File: content/en/docs/ops/telemetry/in-proxy-service-telemetry/index.md [hunk 3]
```diff
@@ -38,6 +38,16 @@ To generate service-level metrics directly in the Envoy proxies, follow these st
     $ kubectl -n istio-system apply -f https://raw.githubusercontent.com/istio/proxy/master/extensions/stats/testdata/istio/metadata-exchange_filter.yaml
     {{< /text >}}
 
+    {{< tip >}}
+    Alternatively, if you have Istio 1.3.x, you can enable metadata exchange filter with WebAssembly sandbox using the
```
Comment (geeknoid) [inline content/en/docs/ops/telemetry/in-proxy-service-telemetry/index.md]:
```suggestion
    Alternatively, if you have Istio 1.3 or later, you can enable the metadata exchange filter with the WebAssembly sandbox using the
```

**Comment by geeknoid** [comment]
@gargnupur @PiotrSikora Should we get this in for 1.4?

**Comment by gargnupur** [comment]
@geeknoid , @PiotrSikora  @mandarjog : I don't think we need this PR as this is for v8 wasm and for 1.4 we are using null wasm only afaik?

**Comment by PiotrSikora** [comment]
@gargnupur that's incorrect. While we ship Telemetry v2 as BETA using NullVm, people can still use it as EXPERIMENTAL using V8, so we need this to be documented.

**Comment by kyessenov** [comment]
@PiotrSikora I don't think we document experimental features in the docs, only something we can fully support. We do need to create a wiki/developer-oriented docs to try out WASM.

**Comment by geeknoid** [comment]
@kyessenov We have several experimental features documented in the doc set. They have [Experimental] in their title, and the content starts with:

{{< boilerplate experimental-feature-warning >}}

which displays a standard warning at the top of the page.

**Comment by gargnupur** [comment]
@PiotrSikora , @kyessenov, @geeknoid  : do we want to get this PR in?
Will have to change it for 1.4 and further as the wasm yaml file is available only in 1.3 branch...

**Comment by PiotrSikora** [comment]
Yes.

**Comment by frankbu** [comment]
Closing, since this is no longer applicable in Istio 1.5+.

**Pull request closed without merging**